static/js/graphs_page.js

```diff
@@ -1990,7 +1992,374 @@ var graphPage = {
+                    maxSimulationTime: 4000
+                }
+            }
+
+            graphPage.applyLayout(newGraphLayout);
+            console.log("after applying layout");
+
+        },
+        flowYChanged : function (checked) {
+            var newGraphLayout;
+
+            //When flow checkbox is checked
+            if (checked)
+            {
+                cola_flow = { axis: "y", minSeparation: 10};
+                $("#input_flow_x").prop("checked", false);
+                newGraphLayout = {
+                    name: 'cola',
+                    nodeSpacing: cola_node_space,
+                    edgeLengthVal: cola_edge_length,
+                    animate: true,
+                    randomize: cola_randomize,
+                    avoidOverlap: cola_avoid_overlap,
+                    fit: cola_fit,
+                    flow: cola_flow,
+                    infinite: cola_infinite,
+                    maxSimulationTime: 4000
+                }
+
+            }
+            //Both x and y are unchecked
+            else
+            {
+                cola_flow = undefined;
+                newGraphLayout = {
+                    name: 'cola',
+                    nodeSpacing: cola_node_space,
+                    edgeLengthVal: cola_edge_length,
+                    animate: true,
+                    randomize: cola_randomize,
+                    avoidOverlap: cola_avoid_overlap,
+                    fit: cola_fit,
+                    flow: cola_flow,
+                    infinite: cola_infinite,
+                    maxSimulationTime: 4000
+                }
+            }
+
+            graphPage.applyLayout(newGraphLayout);
+            console.log("after applying layout");
+
+        },
+        setInputEdgeLength: function () {
+            /*
+             * Updates the text box when the user slides the bar.
+             */
+            if ($("#input_edge").val() < 1) {
+                $("#input_edge").val(1);
+            }
+            if (parseInt($("#input_edge").val()) > 200) {
+                $("#input_edge").val(200);
+
+            }
+            graphPage.colaOption.setBarToValueEdgeLength($("#input_edge"), "slider1");
+            $("#slider1").slider({
+                value: $("#input_edge").val(),
+                max: 200
+            });
+        },
+        setInputNodeSpace: function () {
+            /*
+             * Updates the text box when the user slides the bar.
+             */
+            if ($("#input_node").val() < 1) {
+                $("#input_node").val(1);
+            }
+            if (parseInt($("#input_node").val()) > 50) {
+                $("#input_node").val(50);
+
+            }
+            graphPage.colaOption.setBarToValueNodeSpace($("#input_node"), "slider2");
+            $("#slider2").slider({
+                value: $("#input_node").val(),
+                max: 50
+            });
+        },
+        applyMax: function (graph_layout) {
```

Comment:
Function name is not descriptive enough. What is max ?If possible add link to cola documentation.